modules/search/instant-search/lib/api.js

```diff
@@ -4,13 +4,16 @@
 import fetch from 'unfetch';
 import { encode } from 'qss';
 import { flatten } from 'q-flat';
+import stringify from 'fast-json-stable-stringify';
+import Cache from 'cache';
 
 /**
  * Internal dependencies
  */
 import { getFilterKeys } from './query-string';
 
 const isLengthyArray = array => Array.isArray( array ) && array.length > 0;
+const apiCache = new Cache( 300 * 1000 ); //5 min TTL fully in memory cache
```

Comment:
The comment helps, but could we define some time constants to avoid magic numbers here? 

`const apiCache = new Cache( ( SECONDS_PER_MINUTE * 5 ) * MILLISECONDS_PER_SECOND );`

Not advocating we use a package, but here are some examples:

https://www.npmjs.com/package/time-constants

Reply:
Since we probably won't be doing this often, I just created `FIVE_MINUTES_IN_MILLISECONDS` as a const.